torch/_dynamo/variables/tensor.py

```diff
@@ -586,15 +586,22 @@ def call_method(
 
         from .builder import wrap_fx_proxy
 
```

Comment:
So what I was proposing in the issue description is to remove this code entirely and error if there isn't a handler. This will ensure that we properly handle each method call for tensor variable and guarantee that the return types are what we expect. The change you've made here simply errors earlier, which doesn't solve the issue that we're inserting  arbitrary methods in the graph. Could you try removing this code and seeing which tests fail? this will give us the list of methods we need to add handlers for.

Reply:
Hi @mlazos, Just FYI removed the mentioned code. Please review and let me know if anything else is required.